### ISSUE TITLE
Angular v8 - import 'web-animations-js' polyfill breaks the app

### ISSUE THREAD

**Issue by robisim74** [angular/angular-cli#14518]
# 🐞 Bug report

### Command (mark with an `x`)

```
- [ ] new
- [x ] build
- [ ] serve
- [ ] test
- [ ] e2e
- [ ] generate
- [ ] add
- [ ] update
- [ ] lint
- [ ] xi18n
- [ ] run
- [ ] config
- [ ] help
- [ ] version
- [ ] doc
```

### Is this a regression?

 Yes, the previous version in which this bug was not present was Angular v7

### Description

If I add `import 'web-animations-js';` into `polyfills.ts` , in production it breaks the app. 

## 🔬 Minimal Reproduction

- Create new project using @angular/cli 8.0.0-rc.4
- `npm install --save web-animations-js`
- Enable` import 'web-animations-js';` into` polyfills.ts`
- `ng build --prod`
- Run the app, for example using _http-server_

## 🔥 Exception or Error
<pre><code>

Uncaught TypeError: Cannot set property 'true' of undefined
    at Object.6dTf (polyfills-es2015.c4346c40f18bcedaacdd.js:1)
    at i (runtime-es2015.38622beb49066db5cb2b.js:1)
    at Module.hN/g (polyfills-es2015.c4346c40f18bcedaacdd.js:1)
    at i (runtime-es2015.38622beb49066db5cb2b.js:1)
    at Object.1 (polyfills-es2015.c4346c40f18bcedaacdd.js:1)
    at i (runtime-es2015.38622beb49066db5cb2b.js:1)
    at t (runtime-es2015.38622beb49066db5cb2b.js:1)
    at Array.r [as push] (runtime-es2015.38622beb49066db5cb2b.js:1)
    at polyfills-es2015.c4346c40f18bcedaacdd.js:1
Uncaught Error: In this configuration Angular requires Zone.js
    at new Aa (main-es2015.578a9f97ccae57b370de.js:1)
    at Ya.bootstrapModuleFactory (main-es2015.578a9f97ccae57b370de.js:1)
    at Module.zUnb (main-es2015.578a9f97ccae57b370de.js:1)
    at i (runtime-es2015.38622beb49066db5cb2b.js:1)
    at Object.0 (main-es2015.578a9f97ccae57b370de.js:1)
    at i (runtime-es2015.38622beb49066db5cb2b.js:1)
    at t (runtime-es2015.38622beb49066db5cb2b.js:1)
    at Array.r [as push] (runtime-es2015.38622beb49066db5cb2b.js:1)
    at main-es2015.578a9f97ccae57b370de.js:1
</code></pre>

## 🌍 Your Environment
<pre><code>
Angular CLI: 8.0.0-rc.4
Node: 10.14.2
OS: win32 x64
Angular: 8.0.0-rc.4
... animations, cli, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.800.0-rc.4
@angular-devkit/build-angular     0.800.0-rc.4
@angular-devkit/build-optimizer   0.800.0-rc.4
@angular-devkit/build-webpack     0.800.0-rc.4
@angular-devkit/core              8.0.0-rc.4
@angular-devkit/schematics        8.0.0-rc.4
@ngtools/webpack                  8.0.0-rc.4
@schematics/angular               8.0.0-rc.4
@schematics/update                0.800.0-rc.4
rxjs                              6.4.0
typescript                        3.4.5
webpack                           4.30.0
</code></pre>

**Anything else relevant?**

If I import _zone_ before _web-animation-js_, the app works, but the first error is still present.

**Comment by alan-agius4** [comment]
Related to https://github.com/web-animations/web-animations-js/issues/45

**Comment by bentzibentz** [comment]
Why is this issue closed? The error still exists in v8.0.0. If i import web-animations-js, the application breaks.

**Comment by alan-agius4** [comment]
Hi, for the time being kindly update the dependency version to `github:angular/web-animations-js#release_pr208` as we are waiting for https://github.com/web-animations/web-animations-js/pull/208 to be released.

That said, if you updated to Angular 8 via `ng update` the dependency change should have been done automatically. Otherwise, if it's a new application, I would ask if you really need to add the `web-animations-js` polyfill in the first place. This is typically needed only when using the `AnimationBuilder` API.

Re-opening until we get this solved.

**Comment by dirkluijk** [comment]
Please not that the issue with Zone.js is a separate issue, which can be solved by importing Zone.js differently. See https://github.com/manfredsteyer/ngx-build-plus/issues/4#issuecomment-417912637.

**Comment by Senneseph** [comment]
I believe [#208](https://github.com/web-animations/web-animations-js/pull/208) has been merged / released.

**Comment by mleimer** [comment]
@Senneseph It is merged but not yet released. Latest version is still 2.3.1 from 2 years ago.

**Comment by mgechev** [comment]
Just had a chat with the maintainers. They are working on a release in the next day or two.

**Comment by brookhutchinson** [comment]
Thank you for the update

**Comment by macjohnny** [comment]
for those looking for a workaround to import libraries that are not fixed:
instead of doing `import 'some-library'` in `main.ts` or `polyfills.ts`, add the resulting path to the `scripts` section in your `angular.json`:
```json
{
  "projects": {
    "my-project": {
      "architect": {
        "build": {
          "builder": "@angular-devkit/build-angular:browser",
          "options": {
            "scripts": [
              "node_modules/some-library/some-library.min.js"
            ]
```
this will be bundled in `dist/scripts.js` and in the `dist/index.html` you will have
```html
<script src="scripts.js"></script>
```
without `type="module"`

**Comment by clydin** [comment]
It's also important to note that unless the application is actually using the `AnimationBuilder` class, the animations polyfill is not needed.

**Comment by bfaulk96** [comment]
> Just had a chat with the maintainers. They are working on a release in the next day or two.

"working on a release", they said... "next day or two", they said..

**Comment by Ismaestro** [comment]
Thanks @alan-agius4 I didn't know I don't need to import web-animations, I just thought if I would like to support IE/Edge or Safari I had to import it... Now I can remove it from the project

**Comment by brookhutchinson** [comment]
This issue has been fixed in @angularcli@8.0.3

A heads up to anyone using igniteui-angular npm package... You will need to install igniteui-angular@8.0.0 and web-animations-js@2.3.1 as npm runtime dependencies inside your Angular Project

**Comment by TharakaMadhusanka** [comment]
I also met this issue, so I did uninstall web-animations.js and rebuild and published. Thus this was sorted out. But I am not sure it's the exact solution for this, as web-animations.js is required for IE and Safari browsers, according to the google.

**Comment by clydin** [comment]
It’s generally the correct solution.  The polyfill is only needed for those browsers if the “AnimationBuilder” class is used within the project.  Typical animation use does not require it.

**Comment by alan-agius4** [comment]
web-animations-js published 2.3.2 which includes the fix

**Comment by ViceIce** [comment]
The `blocked` label can be removed.
